src/main.ts

```diff
@@ -96,32 +96,37 @@ const executeDockerCommand = async (dockerCommand: string, continueOnError: bool
 
     const dockerTool: string = await io.which("docker", true);
     var errorStream: string = '';
+    var shouldOutputErrorStream: boolean = false;
     var execOptions: any = {
         outStream: new NullOutstreamStringWritable({ decodeStrings: false }),
         listeners: {
             stdout: (data: any) => console.log(data.toString()), //to log the script output while the script is running.
-            errline: (data: any) => {
-                if (data.toString().trim() === START_SCRIPT_EXECUTION_MARKER) {
-                    errorStream = ''; // Flush the container logs. After this, script error logs will be tracked.
+            errline: (data: string) => {
+                if (!shouldOutputErrorStream) {
+                    errorStream += data + os.EOL;
                 }
                 else {
-                    errorStream += data.toString() + os.EOL;
+                    console.log(data);

```

Comment:
console.log or core.error? so that it shows up in red

Reply:
console.log was preferred as there can be warnings present in error stream, wherein the action still succeeds.